configuration/config-txt/overclocking.md

```diff
@@ -10,31 +10,49 @@ Overclocking and overvoltage will be disabled at runtime when the SoC reaches 85
-| arm_freq_min | Minimum value of arm_freq used for dynamic frequency clocking. The default value is `700` for Pi Zero/Pi 1, `600` for Pi 2/Pi 3. |
-| core_freq_min | Minimum value of `core_freq` used for dynamic frequency clocking. The default value is `250`. On Pi 4B the default is`275` when `hdmi_enable_4kp60` is set|
-| gpu_freq_min | Minimum value of `gpu_freq` used for dynamic frequency clocking. The default value is `250`, or `500` on Pi 4B. |
-| h264_freq_min | Minimum value of `h264_freq` used for dynamic frequency clocking. The default value is `250`, or `500` on Pi 4B. |
-| isp_freq_min | Minimum value of `isp_freq` used for dynamic frequency clocking. The default value is `250`, or `500` on Pi 4B. |
-| v3d_freq_min | Minimum value of `v3d_freq` used for dynamic frequency clocking. The default value is `250`, or `500` on Pi 4B. |
-| sdram_freq_min | Minimum value of `sdram_freq` used for dynamic frequency clocking. The default value is `400`. |
-| over_voltage_min | Minimum value of `over_voltage` used for dynamic frequency clocking. The default value is `0`. |
-| temp_limit | Overheat protection. This sets the clocks and voltages to default when the SoC reaches this value in Celsius.  The default value is `85`. Values over 85 are clamped to 85.|
+| arm_freq_min | Minimum value of arm_freq used for dynamic frequency clocking. |
+| core_freq_min | Minimum value of `core_freq` used for dynamic frequency clocking. On Pi 4B the default is`275` when `hdmi_enable_4kp60` is set|
+| gpu_freq_min | Minimum value of `gpu_freq` used for dynamic frequency clocking.|
+| h264_freq_min | Minimum value of `h264_freq` used for dynamic frequency clocking.|
+| isp_freq_min | Minimum value of `isp_freq` used for dynamic frequency clocking.|
+| v3d_freq_min | Minimum value of `v3d_freq` used for dynamic frequency clocking.|
+| sdram_freq_min | Minimum value of `sdram_freq` used for dynamic frequency clocking.|
+| over_voltage_min | Minimum value of `over_voltage` used for dynamic frequency clocking.|
+| temp_limit | Overheat protection. This sets the clocks and voltages to default when the SoC reaches this value in Celsius. Values over 85 are clamped to 85.|
 | temp_soft_limit | **3A+/3B+ only**. CPU speed throttle control. This sets the temperature at which the CPU clock speed throttling system activates. At this temperature, the clock speed is reduced from 1400Mhz to 1200Mhz.  Defaults to `60`, can be raised to a maximum of `70`, but this may cause instability.
 
+This table gives the default values for the options on various Raspberry Pi Models, all frequencies are stated in MHz.
+
+| Option       | Pi 0/W | Pi1| Pi2 | Pi3   | Pi3B+ | Pi4  |
+| ---          | ---    | --- | --- | ----  | ----- | ---- | 
+| arm_freq     | 1000   | 700 | 900 | 1200  | 1400  | 1500 |
+| gpu_freq     |400/300 | 250 | 250 |400/300|400/300| 500  |  
```

Comment:
The way I understand "Sets `core_freq`, `h264_freq`, `isp_freq`, and `v3d_freq` together; on Pi 3/Pi Zero /Pi Zero W `core_freq` defaults to `400`, while `h264_freq`, `isp_freq`, and `v3d_freq`default to `300`" from the text above, I guess there actually **isn't** a default value for `gpu_freq`; only `core_freq`, `h264_freq`, `isp_freq` & `v3d_freq` have default values? Should `gpu_freq` therefore be removed from this table?
(or have I misunderstood?)